src/isomorphic/ReactDebugTool.js

```diff
@@ -58,17 +58,38 @@ var ReactDebugTool = {
   onSetState() {
     emitEvent('onSetState');
   },
-  onMountRootComponent(internalInstance) {
-    emitEvent('onMountRootComponent', internalInstance);
+  onSetIsTopLevelWrapper(debugID, isTopLevelWrapper) {
+    emitEvent('onSetIsTopLevelWrapper', debugID, isTopLevelWrapper);
   },
-  onMountComponent(internalInstance) {
-    emitEvent('onMountComponent', internalInstance);
+  onSetIsComposite(debugID, isComposite) {
```

Comment:
The main use case so far is ReactPerf only shows composites in the stats. Now that I think of it it probably doesn't matter because we gather their stats independently now. I'll see if I can remove it!